scripts/machine_translation/evaluate_transformer.py

```diff
@@ -215,7 +217,7 @@ def evaluate(args):
     if args.stochastic:
         scorer = BeamSearchScorer(alpha=0.0,
                                   K=0.0,
-                                  temperature=1.0,
+                                  temperature=args.temperature,
```

Comment:
Is it possible to set temperature to None by default and change to 0.5 if it's stochastic beam search?